recipe/meta.yaml

```diff
@@ -34,7 +34,21 @@ about:
   license_family: GPL
```

Comment:
```suggestion
  license_family: LGPL
```

Reply:
Damn, I didn't think to check that. I guess that's wrong in conda-forge as well then...
 
Is that something the linter could check for? i.e. Vet what we have against the `LICENSE` file found from the repo supplied by`dev_url`

Reply:
It could, but it doesn't download upstream code at the moment.